tools/db_bench_tool.cc

```diff
@@ -4645,8 +4695,15 @@ class Benchmark {
       mgr =
           std::make_shared<RoundRobinManager>(GetBuiltinV2CompressionManager());
     } else if (!strcasecmp(FLAGS_compression_manager.c_str(),
-                           "costpredictor")) {
-      mgr = CreateCostAwareCompressionManager();
+                           "autocompalgoselector")) {
+      auto ratelimiter_throughput = FLAGS_rate_limiter_bytes_per_sec;
+      auto io_usage_limit = 0.99 * ratelimiter_throughput;
+      int64_t cpu_usage_limit = 0.9 * kMicrosInSecond;
```

Comment:
0.9 and 0.99 should be part of the configurable so we can run db bench with different budgets and goals.